packages/main/src/ColorPalette.js

```diff
@@ -245,7 +412,7 @@ class ColorPalette extends UI5Element {
 	}
 
 	get displayedColors() {
-		return this.colors.filter(item => item.value).slice(0, 15);
+		return this.getSlottedNodes("colors").filter(item => item.value).slice(0, 15);
```

Comment:
Why can't we read from the slot property anymore?

Reply:
In a meeting it was discussed that this is the better approach.

Reply:
Please use ```colorPaletteColors``` getter